### ISSUE TITLE
Replace `{target-world}` in `execute_commands` action

### ISSUE THREAD

**Issue by BookWyrm114** [PluginBugs/Issues-ItemsAdder#1700]
### Describe the solution you'd like

a new action to allow mythicmob spawning with IA items

### Is your feature request related to a problem?

there's no way to spawn mythicmobs with itemsadder items, even with placeholders because mob spawning commands don't work from console, I've tried everything for weeks, and nothing I've tried has worked

### Describe alternatives you've considered

literally everything, there's seemingly no way to accomplish this.

### Additional context

example config:

items:
  customspawnegg:
    display_name: "customspawnegg"
    permission: test.customspawnegg
    resource:
      material: PAPER
      generate: true
      textures:
      - "customnamespace:item/customspawnegg.png"
    events:
      interact:
        right:
          spawnmythicmob:
            mob: 'mobname'
            atplayerlocation: true
            atlooklocation: false 
               distance: 5

(normally you would not put both 'atplayerlocation' and 'atlooklocation', but I put them here to show them both; at playerlocation would spawn a mob at the player, and atlooklocation would spawn it at the block the player is looking at, using vanilla reach for the maximun distance unless specified otherwise)

**Comment by LoneDev6** [comment]
https://git.mythiccraft.io/mythiccraft/MythicMobs/-/wikis/Commands-and-Permissions#mob-commands

> `/mm mobs spawn -s [mob_name]:<level> <amount> <world,x,y,z>` Spawns mobs with the provided name.

The only missing thing is that ItemsAdder doesn't replace {target-world} in the `execute_commands` action, I can add it.
```yml
        execute_commands:
          cmd:
            command: /mm mobs spawn -s your_mm_mob:1 1 {target-world},{target-x},{target-y},{target-z}
            as_console: true
```

**Comment by LoneDev6** [comment]
This was implemented in `3.1.6`
